src/Core/x/xBehaveGoalSimple.cpp

```diff
@@ -2,44 +2,143 @@
 
 #include <types.h>
 
-// func_8010EB94
-#pragma GLOBAL_ASM("asm/Core/x/xBehaveGoalSimple.s", "xGoalSimple_RegisterTypes__FP8xFactory")
+extern const char lbl_8026A3B0[];
```

Comment:
If this is the string data for this file you should rename the label to something like `xBehaveGoalSimple_strings`

Reply:
Sounds good.